client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -2,38 +2,195 @@
  * External dependencies
  */
 import { bindActionCreators } from 'redux';
-import { localize } from 'i18n-calypso';
+import { translate } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
+import { fetchEmailSettings } from 'woocommerce/state/sites/settings/email/actions';
 import {
-	fetchEmailSettings,
-} from 'woocommerce/state/sites/settings/email/actions';
+	getEmailSettings,
+	areEmailSettingsLoading,
+	areEmailSettingsLoaded,
+} from 'woocommerce/state/sites/settings/email/selectors';
+import CustomerNotification from './components/customer-notification';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import InternalNotification from './components/internal-notification';
+import NotificationsOrigin from './components/notifications-origin';
+import List from 'woocommerce/components/list/list';
+import ListHeader from 'woocommerce/components/list/list-header';
+import ListItemField from 'woocommerce/components/list/list-item-field';
+
+const originNotifications = [
+	{
+		field: 'woocommerce_email_from_address',
+		title: translate( 'From name' ),
+		subtitle: translate( 'Emails will appear in recipients inboxes \'from\' this name.' ),
+	},
+	{
+		field: 'woocommerce_email_from_name',
+		title: translate( 'From address' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **87** times:
`translate( 'Address' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).